packages/notebook/src/actions.tsx

```diff
@@ -68,6 +71,31 @@ const JUPYTER_CELL_MIME = 'application/vnd.jupyter.cells';
  * otherwise specified.
  */
 export
+class NotebookActions {
+  /**
+   * A signal that emits whenever a cell is run.
+   */
+  static get executed(): ISignal<any, { parent: Notebook, child: Cell }> {
```

Comment:
Out of curiosity, do we have any other examples of static signals that are not attached to a class instance?

Reply:
I don't believe we have any other static signals, but since we use the `NotebookActions` namespace basically as though it is a static class, from an API consumer point of view, I thought of it as basically the same thing as the signals on the `CommandRegistry` or any other class that we only instantiate once.

Do you find the pattern problematic or just a curious case?

Reply:
I don't find it problematic, just wondering if we had any other examples of the pattern.